js/src/redux/actions/snippetEditor.js

```diff
@@ -1,6 +1,8 @@
-export const SNIPPET_EDITOR_SWITCH_MODE = "SNIPPET_EDITOR_SWITCH_MODE";
-export const SNIPPET_EDITOR_UPDATE_DATA = "SNIPPET_EDITOR_UPDATE_DATA";
-export const SNIPPET_EDITOR_UPDATE_REPLACEMENT_VARIABLE = "SNIPPET_EDITOR_UPDATE_REPLACEMENT_VARIABLE";
+const prefix = "SNIPPET_EDITOR"
+
+export const SNIPPET_EDITOR_SWITCH_MODE = `${ prefix }_SWITCH_MODE`;
```

Comment:
The thing I like about the prefixing in for example https://github.com/Yoast/yoast-components/blob/eb146b1b8570808192b2e9fc4994d390853bf10e/composites/Plugin/ContentAnalysis/actions/contentAnalysis.js, is that the `const` name becomes shorter, while the value has the prefix. 
That would be `export const SWITCH_MODE = `${ prefix }_SWITCH_MODE`;`

However, that's exactly the part that you haven't copied 😅